challenge.rb

```diff
@@ -22,7 +22,11 @@ def is_of_age? age, minimum_age
 # Remember: `p`, `print`, and `puts` don't return their arguments.
 
 def okay_to_drink? age, drinking_age
-  # Replace this comment with your code.
+  if is_of_age(age, drinking_age)
+     "I'm sorry, #{age} isn't old enough to drink. The minimum age is #{drinking_age}."
+  else
+    puts "Come on in."
+  end
```

Comment:
Based on your answer to Q1, this is correct - nearly. However, the question also says that the string should be _returned_, not printed.